lib/init.coffee

```diff
@@ -78,127 +81,76 @@ module.exports =
       helpers = require 'atom-linter'
       voucher = require 'voucher'
       fs = require 'fs'
-      if @verboseLogging
-        @_log 'requiring modules finished.'
+      @_log 'requiring modules finished.'
 
-    if @verboseLogging
-      @_log 'providing linter, examining javac-callability.'
+    @_log 'providing linter, examining javac-callability.'
 
     grammarScopes: ['source.java']
     scope: 'project'
     lintOnFly: false       # Only lint on save
+
     lint: (textEditor) =>
       filePath = textEditor.getPath()
       wd = path.dirname filePath
       searchDir = @getProjectRootDir() || path.dirname filePath
+
       # Classpath
       cp = ''
 
-      if @verboseLogging
-        @_log 'starting to lint.'
+      # Sourcepath - this is a fair default for many Java projects
+      sp = 'src/main/java/'
```

Comment:
I'm not sure I follow you here. This will only have effect if no sourcepath filename (as specified by the config) is found. I set it to this because most java projects (maven or gradle at least) have this sourcepath.

I put this in an effort to keep as much backwards compatability as possible. It'll be more seamless for current users.